### PR TITLE
Portfolio summary view

### PR DESCRIPTION
**Pull request by JohnPMurray**
This is only a front end view for student portfolio. It is currently only accessible through direct link (localhost:5000/portfolio). It is currently pulling from the shows table to create the portfolios.

### CODE REVIEW

**Comment by JohnPMurray** [comment]
Not sure if we want to wait on the creation of a portfolios table before we merge this, or if we just want to create a separate task to get the correct data once its available

**Comment by johnbyrne727rit** [comment]
I think a separate task for portfolios table is fine, unless you want to take it right away.

File: frontend/src/Student/components/Portfolios.js
Comment (johnbyrne727rit) [inline frontend/src/Student/components/Portfolios.js]: This may be a question for later, but we may want this to always show when there's no open portfolio period, even if they have past portfolios. It may just work like this though, if the past portfolios are queried by user vs open portfolios getting queried by portfolio period.

**Review by johnbyrne727rit — APPROVED** [review]
Looks good to me. I'm glad there's a library that does some of the flipping heavy lifting stuff.

File: frontend/src/Student/components/Portfolios.js
Comment (JohnPMurray) [reply frontend/src/Student/components/Portfolios.js]: Honestly not sure if/when that happens. If it doesn't come up when there are closed periods, then it would also happen for shows. It's not too easy for me to check that since I already have active submission periods open for shows. I'll have to check on it once we have show deletion capabilities (or once I figure out how to do it through the graphql interface) I'll make a bugfix task to check on it.

Comment (JohnPMurray) [reply frontend/src/Student/components/Portfolios.js]: Unless you think we should just get this done now, it probably wouldn't be hard to figure out how to delete stuff through the graphql interface

Comment (JohnPMurray) [reply frontend/src/Student/components/Portfolios.js]: Ignore that I just figured out how to delete shows, its an issue for shows and portfolios.

**Review by moiseslorap — APPROVED** [review]

**Pull request merged**
